config.json

```diff
@@ -12,6 +12,17 @@
       "topics": [
         "println!"
       ]
+    },
+	{
+      "uuid": "c6631a2c-4632-11e8-842f-0ed5f89f718b",
+      "slug": "twofer",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+        "Strings",
+		"match"
```

Comment:
I'd suggest using the topic "string interpolation" here instead of "match": solutions will almost certainly use string interpolation, but I'd expect a good portion of implementers to use an if statement to solve this instead of a match.

Reply:
This is a suggestion, not a requirement; feel free to ignore if desired.